decidim-core/lib/decidim/data_portability_serializers.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module DataPortabilitySerializers
+    autoload :DataPortabilityUserSerializer, "decidim/data_portability_serializers/data_portability_user_serializer"
+    autoload :DataPortabilityUserGroupSerializer, "decidim/data_portability_serializers/data_portability_user_group_serializer"
+    autoload :DataPortabilityConversationSerializer, "decidim/data_portability_serializers/data_portability_conversation_serializer"
+    autoload :DataPortabilityReportSerializer, "decidim/data_portability_serializers/data_portability_report_serializer"
+    autoload :DataPortabilityFollowSerializer, "decidim/data_portability_serializers/data_portability_follow_serializer"
+    autoload :DataPortabilityNotificationSerializer, "decidim/data_portability_serializers/data_portability_notification_serializer"
+    autoload :DataPortabilityIdentitySerializer, "decidim/data_portability_serializers/data_portability_identity_serializer"
+    autoload :DataPortabilityParticipatorySpacePrivateUserSerializer, "decidim/data_portability_serializers/data_portability_participatory_space_private_user_serializer"
+
+    def self.data_entities
```

Comment:
Maybe `Comments` could be put apart to prevent coupling it to core?

Reply:
Any idea on how to do that? Comments haven't `component` or `participatory_space` no?

Reply:
Well, we could add a `Decidim.data_entities` method to register arbitrary entities to.

Reply:
Creating something similiar like `ManifestRegistry` ?